cli/compose/types/types.go

```diff
@@ -191,6 +193,15 @@ type ServiceNetworkConfig struct {
 	Ipv6Address string `mapstructure:"ipv6_address"`
 }
 
+// ServiceSecretConfig is the secret configuration for a service
+type ServiceSecretConfig struct {
```

Comment:
https://github.com/docker/docker/blob/master/api/types/swarm/secret.go#L27 is what `docker service create` is using.